change/@azure-msal-browser-2020-10-07-17-32-50-msal-browser-fetch-headers.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Check for Headers class when configuring network client",
+  "packageName": "@azure/msal-browser",
+  "email": "janutter@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
Could you change this to "none"? I am looking at a way to make an auto config.

Reply:
Why? This is a bugfix.